auth/README.md

```diff
@@ -0,0 +1,69 @@
+# Auth Package
+
+Auth package is used for request authentication and authorization for user to service
+and service to service communication. Auth is used for scenarios that require stricter
```

Comment:
Again here, instead of "Auth is used for," try "Use auth for" or "Reserve auth for." It clarifies who uses auth and when.